tt_metal/impl/program/program.cpp

```diff
@@ -840,7 +841,22 @@ void Program::update_runtime_args_transfer_info(Device* device) {
 
```

Comment:
Does `kernel->cores_with_runtime_args()` need to be updated to not include multicast args?

Reply:
No, that is just for legacy/unicast runtime args to specific cores... with new common args, they are simply multicast to all coreranges that kernel is configured with (`kernel->logical_coreranges()`)